dynacred/src/Contact.ts

```diff
@@ -65,7 +65,7 @@ export class Contact {
     }
 
     set alias(a: string) {
-        if (a) {
+        if (a && this.alias !== a) {
```

Comment:
The idea here is to only update the version of the `Contact` whenever something changes. So for once I like having a setter that will actually `incVersion` if the attribute changes, and just ignores it if it's the same.

Reply:
> The idea here is to only update the version of the `Contact` whenever something changes. So for once I like having a setter that will actually `incVersion` if the attribute changes,

I totally agree: `incVersion` in case of change and not if the same.

> and just ignores it if it's the same.

Which is not what's currently implemented, as when passing an empty string, it doesn't get updated.